Releases Failing

This action is failing when attempting to release via our plugin repo:
https://github.com/jenkinsci/oak9-plugin

The release fails during the `release` step of the CD workflow. This is the error in the log:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy (default-deploy) on project oak9: Failed to deploy artifacts: Could not transfer artifact io.jenkins.plugins:oak9:hpi:1.0.2 from/to maven.jenkins-ci.org (https://repo.jenkins-ci.org/releases/): authentication failed for https://repo.jenkins-ci.org/releases/io/jenkins/plugins/oak9/1.0.2/oak9-1.0.2.hpi, status: 401 Unauthorized -> [Help 1]
```

Being that we don't control the secrets on the repo for authenticating with artifactory, I don't think this is within our control to resolve. Please let me know what we need to do here.

--- a/settings.xml
+++ b/settings.xml
@@ -3,8 +3,14 @@
     <servers>
         <server>
             <id>maven.jenkins-ci.org</id>
-            <username>${env.MAVEN_USERNAME}</username>
-            <password>${env.MAVEN_TOKEN}</password>
+            <configuration>
+                <httpHeaders>
+                    <property>
+                        <name>Authorization</name>
+                        <value>Bearer ${env.MAVEN_TOKEN}</value>
+                    </property>
+                </httpHeaders>
+            </configuration>
         </server>
     </servers>
 </settings>

